From: Oever Gonzalez <notengobattery@gmail.com>
Date: Mon, 7 Jan 2019 01:07:12 +0200
Subject: [PATCH] ath: regd: add extra coutry codes

This patch adds several country codes to the regd.h and regd_common.h
files in order to support devices whose country codes are not present in
the original list. Without this patch, all devices whose manufacturer
programmed any of these code in their EEPROM will not work.

Signed-off-by: Oever Gonzalez <notengobattery@gmail.com>
---

--- a/drivers/net/wireless/ath/regd.h
+++ b/drivers/net/wireless/ath/regd.h
@@ -185,7 +185,9 @@ enum CountryCode {
 	CTRY_UKRAINE = 804,
 	CTRY_UNITED_KINGDOM = 826,
 	CTRY_UNITED_STATES = 840,
+	CTRY_UNITED_STATES2 = 841,
 	CTRY_UNITED_STATES_FCC49 = 842,
+	CTRY_UNITED_STATES3 = 843,
 	CTRY_URUGUAY = 858,
 	CTRY_UZBEKISTAN = 860,
 	CTRY_VENEZUELA = 862,
--- a/drivers/net/wireless/ath/regd_common.h
+++ b/drivers/net/wireless/ath/regd_common.h
@@ -486,6 +486,8 @@ static struct country_code_to_enum_rd al
 	{CTRY_UNITED_KINGDOM, ETSI1_WORLD, "GB"},
 	{CTRY_UNITED_STATES, FCC3_FCCA, "US"},
 	{CTRY_UNITED_STATES, FCC3_FCCA_2, "US"},
+	{CTRY_UNITED_STATES2, FCC3_FCCA, "US"},
+	{CTRY_UNITED_STATES3, FCC3_FCCA, "US"},
 	/* This "PS" is for US public safety actually... to support this we
 	 * would need to assign new special alpha2 to CRDA db as with the world
 	 * regdomain and use another alpha2 */
